tests/mock/Concrete/PlatformOrderDecorator.php

```diff
@@ -0,0 +1,899 @@
+        } catch (\Throwable $e) {
+
+        }
+
+        if (isset($additionalInformation['cc_token_credit_card'])) {
+            $identifier = $additionalInformation['cc_token_credit_card'];
+        }
+
+        if (
+            !empty($additionalInformation['cc_saved_card']) &&
+            $additionalInformation['cc_saved_card'] !== null
+        ) {
+            $identifier = null;
+        }
+
+        if ($identifier === null) {
+            $objectManager = ObjectManager::getInstance();
+            $cardRepo = $objectManager->get(CardsRepository::class);
+            $cardId = $additionalInformation['cc_saved_card'];
+            $card = $cardRepo->getById($cardId);
+
+            $identifier = $card->getCardToken();
+            $customerId = $card->getCardId();
+        }
+
+        $newPaymentData = new \stdClass();
+        $newPaymentData->identifier = $identifier;
+        $newPaymentData->customerId = $customerId;
+        $newPaymentData->brand = $brand;
+        $newPaymentData->installments = $additionalInformation['cc_installments'];
+
+        $newPaymentData->saveOnSuccess =
+            isset($additionalInformation["cc_savecard"]) &&
+            $additionalInformation["cc_savecard"] === '1';
+
+        $amount = str_replace(
+            ['.', ','],
+            "",
+            $additionalInformation["cc_cc_amount"]
+        );
+        $newPaymentData->amount = $moneyService->floatToCents($amount / 100);
+
+        $creditCardDataIndex = AbstractCreditCardPayment::getBaseCode();
+        if (!isset($paymentData[$creditCardDataIndex])) {
+            $paymentData[$creditCardDataIndex] = [];
+        }
+
+        $newPaymentData->customer = $this->extractMultibuyerData(
+            'cc',
+            $additionalInformation
+        );
+
+        $paymentData[$creditCardDataIndex][] = $newPaymentData;
+
+        //boleto
+
+        $newPaymentData = new \stdClass();
+
+        $amount = str_replace(
+            ['.', ','],
+            "",
+            $additionalInformation["cc_billet_amount"]
+        );
+
+        $newPaymentData->amount =
+            $moneyService->floatToCents($amount / 100);
+
+        $boletoDataIndex = BoletoPayment::getBaseCode();
+        if (!isset($paymentData[$boletoDataIndex])) {
+            $paymentData[$boletoDataIndex] = [];
+        }
+
+        $newPaymentData->customer = $this->extractMultibuyerData(
+            'billet',
+            $additionalInformation
+        );
+
+        $paymentData[$boletoDataIndex][] = $newPaymentData;
+    }
+
+    private function extractPaymentDataFromMundipaggBillet(
+        $additionalInformation,
+        &$paymentData,
+        $payment
+    )
+    {
+        $moneyService = new MoneyService();
+        $newPaymentData = new \stdClass();
+        $newPaymentData->amount =
+            $moneyService->floatToCents($this->platformOrder->getGrandTotal());
+
+        $boletoDataIndex = BoletoPayment::getBaseCode();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Mundipagg\Core\Payment\Aggregates\Payments\BoletoPayment' in method 'extractPaymentDataFromMundipaggBillet'.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)